### PR TITLE
Add support for Python 3.10

### PR DESCRIPTION
**Pull request by joshuadavidthomas**
- switches workflow job matrix from rc1 to 3.10 proper, with [passing tests](https://github.com/joshuadavidthomas/django-debug-toolbar/actions)
- updates trove classifiers

### CODE REVIEW

**Comment by joshuadavidthomas** [comment]
Requesting a review because this is my first jazzband repo I have been given the green merge button and I would like another set of eyes before merging.

**Review by auvipy — APPROVED** [review]

**Pull request merged**
